app/auth-portal/src/components/AuthTokenListItem.vue

```diff
@@ -92,4 +142,22 @@ const expiresAt = computed(() =>
 // }
 // /** Name of token to create */
 // const name = ref(authToken.name);
+
+// This pokes the computed values to check if any tokens have expired every 5 seconds
+const EXPIRATION_CHECK_INTERVAL = 5000;
+const checkExpiration = ref();
```

Comment:
checkExpiration isn't used, does it need to be a ref?

Ooh, if you set a `now` variable instead of a tick counter, it gets rid of the need for a bunch of code as well as for the warning itself:

```ts
const now = ref(Date.now())
const expired = computed(() => expiresAt.value?.getTime() <= now.value);

let checkExpiration;
onMounted(() => {
  checkExpiration = setInterval(() => {
    now.value = Date.now(), EXPIRATION_CHECK_INTERVAL));
  }
}
onUnmounted(() => clearInterval(checkExpiration));
```

The current code works fine though, but probably needs a comment in the spot where you ignore the warning.